game/scripts/npc/abilities/phantom_lancer_juxtapose.txt

```diff
@@ -49,12 +49,12 @@
       "07"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "illusion_damage_in_pct"                          "550"
+        "illusion_damage_in_pct"                          "550 550 550 550 500" //OAA
```

Comment:
PL isn't in the game btw.

Reply:
Ye I know lol I just felt like editing code more lol

Reply:
w/o actually changing anything xd